accounts/scwallet/README.md

```diff
@@ -0,0 +1,60 @@
+# Using the smartcard wallet
+
+## Requirements
+
+  * A USB smartcard reader
+  * A keycard that supports the status app
+
+## Preparing the smartcard
+
+  You can use status' [keycard-cli](https://github.com/status-im/keycard-cli) and you should get version 2.1.1 of their [smartcard application](https://github.com/status-im/status-keycard/releases/download/2.1.1/keycard_v2.1.1.cap)
+
+  You also need to make sure that the PCSC daemon is running on your system.
+
+  Then, you can install the application to the card by typing:
+
+  ```
+  keycard install -a keycard_v2.1.cap
+  ```
+
+  Then you can initialize the application by typing:
+
+  ```
+  keycard init
+  ```
+
+  Finally, you need to have the card generate a new master key:
+
+  ```
+  keycard shell <<END
+  keycard-select
+  keycard-set-pairing PAIRING_KEY PAIRING_INDEX
+  keycard-open-secure-channel
+  keycard-verify-pin CARD_PIN
+  keycard-generate-key
```

Comment:
@gballet the generate key command could be dangerous because it deletes the current master key, so in case users have funds in the card, they will lose them. I don't know if this documentation is more for end users or developer so maybe we can add more docs here or we can add a check in the keycard cli so that the user needs to confirm the action.
what do you think?

Reply:
yeah I would be up for adding a check in `keycard-cli`. I have just implemented that check for this PR.